ui/app/AppLayouts/Profile/helpers/ContactDetails.qml

```diff
@@ -0,0 +1,113 @@
+    readonly property bool ensVerified: d.contactDetails.isEnsVerified ?? false
+    readonly property string localNickname: d.contactDetails.localNickname ?? ""
+    readonly property string alias: d.contactDetails.alias ?? ""
+    readonly property string icon: d.contactDetails.icon ?? ""
+    readonly property int colorId: d.contactDetails.colorId ?? 0
+    readonly property var colorHash: d.contactDetails.colorHash ?? []
+    readonly property int onlineStatus: d.contactDetails.onlineStatus ?? Constants.onlineStatus.inactive
+    readonly property bool isContact: d.contactDetails.isContact ?? false
+    readonly property bool isCurrentUser: d.contactDetails.isCurrentUser ?? false
+    readonly property bool isVerified: d.contactDetails.isVerified ?? false
+    readonly property bool isUntrustworthy: d.contactDetails.isUntrustworthy ?? false
+    readonly property bool isBlocked: d.contactDetails.isBlocked ?? false
+    readonly property int contactRequestState: d.contactDetails.contactRequest ?? Constants.ContactRequestState.None
+    readonly property int incomingVerificationStatus: d.contactDetails.incomingVerificationStatus ?? Constants.verificationStatus.unverified
+    readonly property int outgoingVerificationStatus: d.contactDetails.outgoingVerificationStatus ?? Constants.verificationStatus.unverified
+    readonly property string defaultDisplayName: d.contactDetails.defaultDisplayName ?? ""
+    readonly property string optionalName: d.contactDetails.optionalName ?? ""
+    readonly property int lastUpdated: d.contactDetails.lastUpdated ?? 0
+    readonly property int lastUpdatedLocally: d.contactDetails.lastUpdatedLocally ?? 0
+    readonly property string thumbnailImage: d.contactDetails.thumbnailImage ?? ""
+    readonly property string largeImage: d.contactDetails.largeImage ?? ""
+    readonly property bool isContactRequestReceived: d.contactDetails.isContactRequestReceived ?? false
+    readonly property bool isContactRequestSent: d.contactDetails.isContactRequestSent ?? false
+    readonly property bool isSyncing: d.contactDetails.isSyncing ?? false
+    readonly property bool removed: d.contactDetails.isRemoved ?? false
+    readonly property int trustStatus: d.contactDetails.trustStatus ?? Constants.trustStatus.unknown
+    readonly property string bio: d.contactDetails.bio ?? ""
+
+    // Backwards compatibility properties - Don't use in new code
+    // TODO: #14965 - Try to remove these properties
+    readonly property string name: ensName
+    readonly property int verificationStatus: outgoingVerificationStatus
+
+    // Extra properties provided by getContactDetailsAsJson, not available in the model
+    // TODO: #14964 - Review all the model rolenames and fill the rest of the properties with data from the model
+    //readonly property int verificationStatus: d.contactDetails.verificationStatus ?? Constants.verificationStatus.unverified
+    //readonly property var socialLinks: d.contactDetails.socialLinks ?? []
```

Comment:
This has been removed imo (or should be)

Reply:
I also think so.Let's see in #14964 if we need these roles in the model or not.